docs/pages/push-notifications/faq.mdx

```diff
@@ -107,6 +107,18 @@ Setting the priority to `high` gives your notification the greatest likelihood t
 
 When your push notification credentials have expired, run `eas credentials`, choose iOS and a build profile, then remove your push notification key and generate a new one.
 
+### Error no valid aps-environment entitlement string found for application
+
+This error occurs if you haven't set up a push notification key for your project. To check, go to the [Project Credentials page](https://expo.dev/accounts/[account]/projects/[project]/credentials/ios).
+
+To generate a new push notification key, trigger a new build by running:
+
+```bash
```

Comment:
We prefer using the Terminal component for commands: https://github.com/expo/expo/blob/main/docs/README.md#use-terminal-component-for-shell-commands-snippets